### ISSUE TITLE
[0.2.3] Linking fails on OS X with Rust nightly

### ISSUE THREAD

**Issue by jmacdonald** [gchp/termbox-sys#6]
As a dependency of rustbox, I'm seeing the following output when building a project:

```
error: linking with `cc` failed: exit code: 1
note: "cc" "-m64" "-L" "/usr/local/lib/rustlib/x86_64-apple-darwin/lib" "-o" "/Users/jmacdonald/projects/rust/amp/target/debug/amp" "/Users/jmacdonald/projects/rust/amp/target/debug/amp.o" "-Wl,-force_load,/usr/local/lib/rustlib/x86_64-apple-darwin/lib/libmorestack.a" "-Wl,-dead_strip" "-nodefaultlibs" "/Users/jmacdonald/projects/rust/amp/target/debug/deps/librustbox-a94cefed7498260c.rlib" "/Users/jmacdonald/projects/rust/amp/target/debug/deps/libscribe-2e2af195f8d5c571.rlib" "/Users/jmacdonald/projects/rust/amp/target/debug/deps/libtermbox_sys-ede589403b9421d2.rlib" "/Users/jmacdonald/projects/rust/amp/target/debug/deps/libbitflags-518ea12e21428edd.rlib" "/Users/jmacdonald/projects/rust/amp/target/debug/deps/libluthor-50043075bc9a0ab9.rlib" "/Users/jmacdonald/projects/rust/amp/target/debug/deps/libpad-548f040d9866039b.rlib" "/usr/local/lib/rustlib/x86_64-apple-darwin/lib/libstd-4e7c5e5c.rlib" "/usr/local/lib/rustlib/x86_64-apple-darwin/lib/libcollections-4e7c5e5c.rlib" "/usr/local/lib/rustlib/x86_64-apple-darwin/lib/libunicode-4e7c5e5c.rlib" "/usr/local/lib/rustlib/x86_64-apple-darwin/lib/librand-4e7c5e5c.rlib" "/usr/local/lib/rustlib/x86_64-apple-darwin/lib/liballoc-4e7c5e5c.rlib" "/usr/local/lib/rustlib/x86_64-apple-darwin/lib/liblibc-4e7c5e5c.rlib" "/usr/local/lib/rustlib/x86_64-apple-darwin/lib/libcore-4e7c5e5c.rlib" "-L" "/Users/jmacdonald/projects/rust/amp/target/debug" "-L" "/Users/jmacdonald/projects/rust/amp/target/debug/deps" "-L" "/Users/jmacdonald/projects/rust/amp/target/debug/build/termbox-sys-ede589403b9421d2/out/lib" "-L" "/usr/local/lib/rustlib/x86_64-apple-darwin/lib" "-L" "/Users/jmacdonald/projects/rust/amp/.rust/lib/x86_64-apple-darwin" "-L" "/Users/jmacdonald/projects/rust/amp/lib/x86_64-apple-darwin" "-ltermbox:static" "-lSystem" "-lpthread" "-lc" "-lm" "-lcompiler-rt"
note: ld: warning: directory not found for option '-L/Users/jmacdonald/projects/rust/amp/lib/x86_64-apple-darwin'
ld: library not found for -ltermbox:static
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by jmacdonald** [comment]
Fixed as a result of #7. Thanks for merging that. :wink:

**Comment by gchp** [comment]
Cool, no prob :D
